README.md

```diff
@@ -14,11 +16,13 @@ If you new to Java, or to Software Engineering, you should simply try to downloa
 
 Otherwise, I highly recommend you to manage your Java installations using the awesome https://sdkman.io/
 
-## Quickstart
+### Access Nubank's Github 
+
+Once you have the required software installed (see #requirements) ensure you have access to Github and create a personal access token.
 
-Once you have the required software installed (see #requirements), you got follow a couple of steps:
+## Step to run the tool
 
-### 1 - Create your personal access token in Github
+### 1 - Create your personal access token in Github (optional)
```

Comment:
It's not an optional step. I changed it to become mandatory since the user experience is terrible when fetching data from popular repositories due to the rate-limiting imposed by GH. 

```suggestion
### 1 - Create your personal access token in Github
```